docs/reference/rest-api/security/bulk-create-roles.asciidoc

```diff
@@ -0,0 +1,342 @@
+[role="xpack"]
+[[security-api-bulk-put-role]]
+=== Create or update roles in Bulk API
```

Comment:
```suggestion
=== Create or update roles in bulk API
```

or (preferred):

```suggestion
=== Bulk create or update roles API
```

to match [the other bulk security api](https://elasticsearch_bk_109339.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/security-api-bulk-update-api-keys.html)

Reply:
Great! Thanks!